firebase-test-lab/trigger_ftl_tests.py

```diff
@@ -0,0 +1,346 @@
+# coding: utf-8
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+r"""Tool for sending mobile testapps to Firebase Test Lab for testing.
+
+Requires Cloud SDK installed with gsutil. Can be checked as follows:
+  gcloud --version
+
+This tool will use the GCS storage bucket, thus must be authorized first.
+Usage:
+
+  python trigger_ftl_tests.py --testapp_dir ~/testapps --test_type xctest
+
+This will recursively search testapps under dir "~/testapps" for apks, ipas,
+and zips, send them to FTL, and generate test summary: 
+
+Note: For android instrumentation testapps, please compress both apks in a
+zip file and make sure the test apk name contains string "test". e.g. 
```

Comment:
Because the input is testapp_dir, thus it's a headache to pair the apps and distinguish '--app' and '--test'.
Is better to have some rules here. What's your idea?
Also, in the ReadMe, will suggest users to use yaml file as input in this case.

Reply:
We surely can set whatever rule we want, but it is also up to users whether they want to play along with our rules. If it is too much inconvenient, (and I feel it is a bit trouble to require users to zip beforehand), users will just go away. Let's discuss this more offline.

Reply:
The same problem goes with XCTest: https://firebase.google.com/docs/test-lab/ios/run-xctest#package-app